app/views/finders/show.html.erb

```diff
@@ -73,20 +73,23 @@
           <div aria-live='assertive' id='js-search-results-info' data-module="remove-filter">
             <% if finder.show_generic_description? %>
```

Comment:
Not part of this change, but we don't use `show_generic_description?` anymore [since it was removed from the business finder](
https://github.com/alphagov/govuk-app-deployment-secrets/pull/45/files), so it may be worth removing this now

Reply:
Removed here https://github.com/alphagov/finder-frontend/pull/856/commits/3a90d6f83ebf938a512521f9ff1cf8ebfdcac64c